### DISCUSSION TITLE
Net Cash Flow

### DISCUSSION THREAD

**Discussion by fib1985** [firefly-iii/firefly-iii#9172] [General]
Hi all!

It is possible to view or generate a report that include the **net cash flow** for each month or whatever timestamp? Soomething like this should be very useful! thanks
![netcashflow](https://github.com/user-attachments/assets/a4414ce4-583d-4b8e-84ea-b68d54c4d436)

**Comment by JC5** [comment]
No, not yet.

**Reply by fib1985** [reply]
Hi James,
thanks for the feedback! I hope that this crucial feature will be implemented in the next future. For now I continue to enjoy this awesome application! Have a good day!

**Reply by quovibe** [reply]
Hi @JC5,

there are any news on this? :) thank you
